### ISSUE TITLE
Warning: Missing Node Types

### ISSUE THREAD

**Issue by Black911X** [TemryL/ComfyUI-IDM-VTON#83]
Install from manager, install manually, try fix, doesn't help

![1](https://github.com/user-attachments/assets/138fd1c1-4435-4888-9d46-2168f655078c)
![2](https://github.com/user-attachments/assets/36d97b42-62b8-4ffb-a371-1fc24d2c8303)
![3](https://github.com/user-attachments/assets/717d79a2-1b4f-4c99-813e-de8a2138d095)

Python 3.10.11

absl-py                   2.1.0
accelerate                0.30.0
addict                    2.4.0
aiohappyeyeballs          2.4.3
aiohttp                   3.10.10
aiosignal                 1.3.1
albucore                  0.0.16
albumentations            1.4.15
annotated-types           0.7.0
antlr4-python3-runtime    4.9.3
anyio                     4.6.2.post1
arrow                     1.3.0
async-timeout             4.0.3
attrs                     24.2.0
av                        12.3.0
beautifulsoup4            4.12.3
binaryornot               0.4.4
bitsandbytes              0.42.0
boto3                     1.35.45
botocore                  1.35.45
cachetools                5.5.0
certifi                   2024.7.4
cffi                      1.17.1
cfgv                      3.4.0
chardet                   5.2.0
charset-normalizer        3.3.2
click                     8.1.7
clip-interrogator         0.6.0
cmake                     3.30.5
color-matcher             0.5.0
colorama                  0.4.6
coloredlogs               15.0.1
colorlog                  6.8.2
colour-science            0.4.6
contourpy                 1.3.0
cookiecutter              2.6.0
cryptography              43.0.3
cssselect2                0.7.0
cstr                      0.1.0
ctranslate2               4.4.0
cycler                    0.12.1
ddt                       1.7.2
deepdiff                  8.0.1
Deprecated                1.2.14
diffusers                 0.27.2
dill                      0.3.9
distlib                   0.3.9
distro                    1.9.0
docstring_parser          0.16
docutils                  0.21.2
easydict                  1.13
einops                    0.8.0
eval_type_backport        0.2.0
exceptiongroup            1.2.2
fairscale                 0.4.13
fastapi                   0.115.2
faster-whisper            1.0.3
ffmpy                     0.3.0
filelock                  3.15.4
flatbuffers               24.3.25
flet                      0.24.1
flet-core                 0.24.1
flet-runtime              0.24.1
fonttools                 4.54.1
frozenlist                1.4.1
fsspec                    2024.6.1
ftfy                      6.3.0
fvcore                    0.1.5.post20221221
gdown                     5.2.0
gitdb                     4.0.11
GitPython                 3.1.43
h11                       0.14.0
httpcore                  1.0.6
httptools                 0.6.4
httpx                     0.27.2
huggingface-hub           0.26.0
humanfriendly             10.0
hydra-core                1.3.2
identify                  2.6.1
idna                      3.8
imageio                   2.36.0
imageio-ffmpeg            0.5.1
img2texture               1.0.6
importlib_metadata        8.5.0
iniconfig                 2.0.0
instructor                1.6.3
iopath                    0.1.10
jax                       0.4.34
jaxlib                    0.4.34
Jinja2                    3.1.4
jiter                     0.5.0
jmespath                  1.0.1
joblib                    1.4.2
json_repair               0.30.0
jsonschema                4.23.0
jsonschema-specifications 2024.10.1
kaldi-native-fbank        1.20.0
kiwisolver                1.4.7
kornia                    0.7.3
kornia_rs                 0.1.5
lark-parser               0.12.0
lazy_loader               0.4
lightning-utilities       0.11.8
llvmlite                  0.43.0
loralib                   0.1.2
lxml                      5.3.0
manifold3d                2.5.1
mapbox_earcut             1.0.2
markdown-it-py            3.0.0
MarkupSafe                3.0.2
matplotlib                3.9.2
matrix-client             0.4.0
mdurl                     0.1.2
mediapipe                 0.10.14
ml_dtypes                 0.5.0
mpmath                    1.3.0
mss                       9.0.2
multidict                 6.1.0
natsort                   8.4.0
networkx                  3.4.2
nodeenv                   1.9.1
numba                     0.60.0
numpy                     1.26.4
oauthlib                  3.2.2
omegaconf                 2.3.0
onnxruntime               1.19.2
onnxruntime-gpu           1.19.2
open_clip_torch           2.27.1
openai                    1.52.0
opencv-contrib-python     4.10.0.84
opencv-python             4.10.0.84
opencv-python-headless    4.10.0.84
opt_einsum                3.4.0
orderly-set               5.2.2
packaging                 23.2
piexif                    1.1.3
pilgram                   1.2.1
pillow                    11.0.0
pip                       24.2
pixeloe                   0.0.10
platformdirs              4.3.6
pluggy                    1.5.0
pooch                     1.8.2
portalocker               2.10.1
pre_commit                4.0.1
pretty-errors             1.2.25
propcache                 0.2.0
protobuf                  4.25.5
psutil                    6.1.0
py-cpuinfo                9.0.0
pycocoevalcap             1.2
pycocotools               2.0.8
pycollada                 0.8
pycparser                 2.22
pydantic                  2.9.2
pydantic_core             2.23.4
PyGithub                  2.4.0
Pygments                  2.18.0
PyJWT                     2.9.0
PyMatting                 1.1.12
PyNaCl                    1.5.0
pynvml                    11.5.3
pyOpenSSL                 24.2.1
pyparsing                 3.2.0
pypng                     0.20220715.0
pyreadline3               3.5.4
PySocks                   1.7.1
pytest                    8.3.3
python-dateutil           2.9.0.post0
python-dotenv             1.0.1
python-slugify            8.0.4
pytorch-lightning         2.4.0
pywin32                   308
PyYAML                    6.0.2
qrcode                    7.4.2
referencing               0.35.1
regex                     2024.9.11
rembg                     2.0.59
repath                    0.9.0
reportlab                 4.2.5
requests                  2.32.3
rich                      13.8.1
rpds-py                   0.20.0
Rtree                     1.3.0
s3transfer                0.10.3
safetensors               0.4.5
scenedetect               0.6.4
scikit-image              0.24.0
scikit-learn              1.5.2
scipy                     1.13.0
segment-anything          1.0
sensevoice-onnx           1.1.0
sentencepiece             0.2.0
setuptools                65.0.0
shapely                   2.0.6
shellingham               1.5.4
simpleeval                1.0.0
six                       1.16.0
smmap                     5.0.1
sniffio                   1.3.1
sounddevice               0.5.1
soundfile                 0.12.1
soupsieve                 2.6
starlette                 0.40.0
svg.path                  6.3
svglib                    1.5.1
swarm                     0.1.0
sympy                     1.13.1
tabulate                  0.9.0
tenacity                  9.0.0
termcolor                 2.5.0
text-unidecode            1.3
threadpoolctl             3.5.0
tifffile                  2024.9.20
timm                      1.0.11
tinycss2                  1.3.0
tokenizers                0.19.1
tomli                     2.0.2
torch                     2.5.0
torchaudio                2.5.0
torchmetrics              1.4.0
torchvision               0.20.0
tqdm                      4.66.4
transformers              4.40.2
transparent-background    1.3.3
trimesh                   4.5.0
typer                     0.12.5
types-python-dateutil     2.9.0.20241003
typing_extensions         4.12.2
urllib3                   1.26.20
uvicorn                   0.32.0
vhacdx                    0.0.8.post1
virtualenv                20.27.0
watchdog                  4.0.2
watchfiles                0.24.0
wcwidth                   0.2.13
webcolors                 24.8.0
webencodings              0.5.1
websockets                13.1
wget                      3.2
wrapt                     1.16.0
xatlas                    0.0.9
xxhash                    3.5.0
yacs                      0.1.8
yapf                      0.40.2
yarl                      1.15.5
zipp                      3.20.2

**Comment by RowlData** [comment]
I'm having the same issue. Any solutions?
